.github/actions/on_host_tests/action.yaml

```diff
@@ -55,8 +54,11 @@ runs:
         failed_suites=""
         cd "${test_dir}"
 
-        for test_binary_path in $(cat out/${{ matrix.platform}}_${{ matrix.config }}/test_targets.json | jq -cr '.executables | join(" ")'); do
+        for test_binary_path in $(cat out/${{ matrix.platform }}_${{ matrix.config }}/test_targets.json | jq -cr '.executables | join(" ")'); do
           test_binary=$(basename "${test_binary_path}")
+          if [[ "${{ matrix.platform }}" == *evergreen* ]]; then
+            test_binary="nplb_loader"
```

Comment:
@oxve I hardcoded a number of things in this PR, because right now the github action files make the assumption that the GN target name will be the same as the executable name which does not hold for evergreen (unless we make further assumptions for evergreen about the naming scheme). We can discuss this in our writeup doc, but I'm thinking about doing this for now just to get nplb running, wdyt?